Kanvi/remove files (#271)
tensorflow/ngraph-bridge: diagnostics/README.md

| `NGRAPH_TF_DUMP_DECLUSTERED_GRAPHS=1` | Dump graphs with final clusters assigned. Use this to view TF computation graph with colored nodes indicating clusters|

> |

Author: ```suggestion
```

## Protobuf visualization

> The [tf2ngraph] script can be used to convert tensorflow graph to an ngraph enabled graph

Reviewer: This is not very user friendly. Explain the use case in more detailed and provide better example.

and dump it as a protobuf (`pb`).

> Tensorflows' `import_pb_tensorboard.py` script can then be used to view the

Reviewer 2: ```suggestion
Tensorflow's `import_pb_tensorboard.py` script can then be used to view the
```

## Protobuf visualization

> The [tf2ngraph] script can be used to convert tensorflow graph to a ngraph graph

Reviewer 2: [tf2ngraph] 's link is not provided at the end of the file. So in the markdown rendering it does not show up as a link, but simply as `[tf2ngraph`

Author: Should it be at the end of the file or end of line ?

Reviewer 2: end of file. the .md file has a bunch of links at EOF. so if u add an entry there,  [tf2ngraph] will show up as a link

> The [tf2ngraph] script can be used to convert tensorflow graph to a ngraph graph

Reviewer 2: ```suggestion
The [tf2ngraph] script can be used to convert tensorflow graph to an ngraph enabled graph
```

and dump it as a protobuf (`pb`).(https://github.com/tensorflow/ngraph-bridge/blob/master/tools/tf2ngraph.py)
Tensorflows' `import_pb_tensorboard.py` script can then be used to view the

> ngraph graph on `tensorboard`.

Reviewer 2: ```suggestion
dumped graph on `tensorboard`.
```

Reviewer 2: tf2ngraph does not dump an ngraph graph. it dumps a TF graph, which is ngraph ready

### Examples usage

* python tf2ngraph --input_pbtxt test_axpy.pbtxt --output_nodes add --output_pb axpy_ngraph.pb --ng_backend CPU

> * Add `import ngraph_bridge` to `import_pb_tensorboard.py` script

Reviewer 2: ```suggestion
* Add `import ngraph_bridge` to `import_pb_to_tensorboard.py` script
```

> * python import_pb_to_tensorboard.py --model_dir axpy_ngraph.pb --log_dir test/

Reviewer 2: Add TF link to `import_pb_to_tensorboard.py`

## Protobuf visualization

The [tf2ngraph] script can be used to convert tensorflow graph to an ngraph enabled graph
and dump it as a protobuf (`pb`).
Tensorflows' `import_pb_tensorboard.py` script can then be used to view the
dumped graph on `tensorboard`.

### Example usage

* python tf2ngraph --input_pbtxt test_axpy.pbtxt --output_nodes add --output_pb axpy_ngraph.pb --ng_backend CPU
* Add `import ngraph_bridge` to [import_pb_to_tensorboard.py] script

> * python import_pb_to_tensorboard.py --model_dir axpy_ngraph.pb --log_dir test/

Reviewer 2: ```suggestion
* `python import_pb_to_tensorboard.py --model_dir axpy_ngraph.pb --log_dir test/`
```

> * Launch Tensorboard by pointing it to the log directory : tensorboard --logdir=test/

Reviewer 2: ```suggestion
* Launch Tensorboard by pointing it to the log directory : `tensorboard --logdir=test/`
```
